futures-async-runtime/src/lib.rs

```diff
@@ -6,11 +7,15 @@
 
 macro_rules! if_nightly {
     ($($i:item)*) => ($(
-        #[cfg(feature = "nightly")]
+        #[cfg(all(feature = "nightly", feature = "std"))]
```

Comment:
This looks wrong; `if_nightly` should not check for `std` - maybe rename to `if_nightly_and_std` if you actually need that?

Reply:
This is just a temporary change to get `futures` compiling, I've got another branch that will get `futures-async-runtime` actually working with `no_std` almost ready to go. But sure, I can rename the macro in the meantime.